man/systemd.netdev.xml

```diff
@@ -911,6 +916,23 @@
           <para>Accepts the same key as <literal>[FooOverUDP]</literal></para>
         </listitem>
       </varlistentry>
+      <varlistentry>
+        <term><varname>Sequence=</varname></term>
+        <listitem>
+          <para>Takes a boolean value. If set to yes, then packets are serialized. Only applies for ERSPAN tunnel.
+          Defaults to <literal>unset</literal>.
```

Comment:
`<literal>` is not applicable here. So maybe "Defaults to unset, which means to use the kernel default.".